content/en/docs/collector/scaling.md

```diff
@@ -171,7 +171,7 @@ kind: Pod
 metadata:
   name: my-microservice
   annotations:
-    sidecar.opentelemetry.io/inject: "true"
+    sidecar.opentelemetry.io/inject: 'true'
```

Comment:
@TylerHelmuth - can we drop the quotes here?

Reply:
No, but single quotes work.